autoupgrade.php

```diff
@@ -185,4 +185,34 @@ public function trans($id, array $parameters = [], $domain = null, $locale = nul
 
         return $translator->trans($id, $parameters);
     }
+
+    /**
+     * Only available from PS8.
+     *
+     * @param array{links: \PrestaShop\PrestaShop\Core\Action\ActionsBarButtonsCollection} $params
+     *
+     * @return void
+     */
+    public function hookDisplayBackOfficeEmployeeMenu(array $params)
+    {
+        if (
+            !class_exists(\PrestaShop\PrestaShop\Core\Action\ActionsBarButtonsCollection::class)
+            || !class_exists(\PrestaShop\PrestaShop\Core\Action\ActionsBarButton::class)
+            || !($params['links'] instanceof \PrestaShop\PrestaShop\Core\Action\ActionsBarButtonsCollection)
+        ) {
+            return;
+        }
+
+        $params['links']->add(
+            new \PrestaShop\PrestaShop\Core\Action\ActionsBarButton(
+                __CLASS__,
+                [
+                    'link' => 'https://build.prestashop-project.org/tag/releases/',
```

Comment:
Do you think we should add this link to the rest in `classes/DocumentationLinks.php`?